src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -317,50 +321,78 @@ def _instrument_layer(self, layer, config):
         if 'weight' in config["quant_types"]:
             if not _check_weight(layer.module):
                 _logger.warning('Module %s does not have parameter "weight"', layer.name)
+            else:
+                layer.module.register_parameter("old_weight", torch.nn.Parameter(layer.module.weight))
```

Comment:
can you unify the use of punctuation? such as `"` and `'`

Reply:
fixed